### PR TITLE
fix integer bug

### PR DESCRIPTION
**Pull request by lucas7788**

### CODE REVIEW

**Comment by WyattMufson** [comment]
What bug does this fix? This doesn't look like it does anything.

File: contract.py [hunk 1]
```diff
@@ -100,14 +100,14 @@ def decimals():
     """
     Returns the amount of decimals of the token
     """
-    return DECIMALS
+    return DECIMALS + 0
```
Comment (WyattMufson) [inline contract.py:103]: Adding `0` to `DECIMALS` does nothing. `DECIMALS` is `8` and adding `0` does not change it's value

Comment (lucas7788) [reply contract.py:103]: +0 will change the data type to integer type, which is very important when calling across contracts.

Comment (WyattMufson) [reply contract.py:103]: neovm does not support floating point numbers, so the number is an integer by default

**Pull request closed without merging**